business-registry-dashboard/app/locales/en-CA.ts

```diff
@@ -223,7 +224,7 @@ export default {
         },
         400: {
           title: 'Error Adding Name Request',
-          description: "We couldn't find a name request associated with the phone number or email address you entered. Please try again."
+          description: "The email or phone number you entered doesn't match with the name request application."
```

Comment:
Did this come from UX team?

This is inconsistent with line 507, below, which calls it a "Name Request".

Reply:
Yep, this was part of the new figma model that was from the UX team:
![image](https://github.com/user-attachments/assets/70143383-037a-4f61-8bc2-c07b418c1e5a)

Reply:
Well, ask them about the mismatch with line 507 whether they still want it this way.

OK to resolve this conversation.

Reply:
So Sev, I finally got an answer back from our UX folks on this. I changed the wording of this to: The email or phone number you entered doesn't match the application for your Name Request.